containers/docker-compose.yaml

```diff
@@ -138,12 +140,22 @@ services:
       - GDS_BFF_CONSOLE_LOG=true
       - GDS_ADMIN_ALLOW_ORIGINS=http://localhost:3000
       - GDS_ADMIN_COOKIE_DOMAIN=localhost
-      - GDS_BFF_TESTNET_INSECURE=true
-      - GDS_BFF_TESTNET_ENDPOINT=gds:4433
-      - GDS_BFF_TESTNET_TIMEOUT=5s
-      - GDS_BFF_MAINNET_INSECURE=true
-      - GDS_BFF_MAINNET_ENDPOINT=gds-mainnet:8433
-      - GDS_BFF_MAINNET_TIMEOUT=5s
+      - GDS_BFF_TESTNET_DIRECTORY_INSECURE=true
+      - GDS_BFF_TESTNET_DIRECTORY_ENDPOINT=gds:4433
+      - GDS_BFF_TESTNET_DIRECTORY_TIMEOUT=5s
+      - GDS_BFF_TESTNET_ADMIN_ENDPOINT=http://gds:4434
+      - GDS_BFF_TESTNET_ADMIN_AUDIENCE=http://localhost:4433
+      - GDS_BFF_TESTNET_ADMIN_TOKEN_KEYS=1y9jUjVqRqRJaiKsNqrZmQkhTqe:/data/creds/current_token_key.pem,1y9jTDgrljWl0iVUOQBcCev7rBG:/data/creds/rotated_token_key.pem
+      - GDS_BFF_TESTNET_MEMBERS_INSECURE=true
+      - GDS_BFF_TESTNET_MEMBERS_ENDPOINT=gds:4435
+      - GDS_BFF_MAINNET_DIRECTORY_INSECURE=true
+      - GDS_BFF_MAINNET_DIRECTORY_ENDPOINT=gds-mainnet:8433
+      - GDS_BFF_MAINNET_DIRECTORY_TIMEOUT=5s
+      - GDS_BFF_MAINNET_ADMIN_ENDPOINT=http://gds:8434
+      - GDS_BFF_MAINNET_ADMIN_AUDIENCE=http://localhost:8433
+      - GDS_BFF_MAINNET_ADMIN_TOKEN_KEYS=1y9jUjVqRqRJaiKsNqrZmQkhTqe:/data/creds/current_token_key.pem,1y9jTDgrljWl0iVUOQBcCev7rBG:/data/creds/rotated_token_key.pem
+      - GDS_BFF_MAINNET_MEMBERS_INSECURE=true
+      - GDS_BFF_MAINNET_MEMBERS_ENDPOINT=gds-mainnet:8435
```

Comment:
Thank you for updating these! @masskoder this should fix the issue you were having.